package.json

```diff
@@ -1089,7 +1089,17 @@
         "visible": true
       },
       {
-        "permissionName": "ui-users.patron-pre-registrations-view",
+        "permissionName": "ui-users.patron-pre-registrations.execute",
+        "displayName": "Users: Can merge patron preregistration data",
+        "description": "Edits existing user or creates new user, based on pre-registration record",
+        "subPermissions": [
```

Comment:
Is it expected, that a user with this perm can't see staging users? Should it include `ui-users.patron-pre-registrations.view`?

Reply:
As per our PO, yes, it is expected.